### ISSUE TITLE
Error with Gtburst and LAT transient builder plugin

### ISSUE THREAD

**Issue by ssajjadak** [threeML/threeML#598]
Hello,

I am following the  LAT transient builder documentation: https://threeml.readthedocs.io/en/stable/md_docs/slow_execute/LAT_Transient_Builder_Example.html for GRB 190114C.

So far, I have managed to carry out all the steps till the likelihood analysis `LAT_observations = analysis_builder.run(include_previous_intervals=True)`. I have not made any changes to the code in the documentation.

The likelihood step runs well till the call to gtsrcprob is made. After that, I get the following error:
`Caught N10Likelihood9ExceptionE at the top level: Event::diffuseResponse: `

I recall that with fermitools, this type of error (Caught N10Likelihood9ExceptionE ...) usually occurs when there is a discrepancy in the IRFs or other settings used with the different tools. Looking closely at the log,  it seems to me that problem occurs because gtsrcprob is called with a different IRF than the previous steps. Here is the line for the gtsrcprob.

`time -p gtsrcprob evfile=/home/sajjad/SOURCES_ASTRO/GRBs/THREE_ML/2022_10/interval2.6996-3.6358/gll_ft1_tr_bn190114873_v00_filt.fit evtable="EVENTS" scfile=/home/sajjad/SOURCES_ASTRO/GRBs/THREE_ML/2022_10/bn190114873/gll_ft2_tr_bn190114873_v00.fit sctable="SC_DATA" outfile=/home/sajjad/SOURCES_ASTRO/GRBs/THREE_ML/2022_10/interval2.6996-3.6358/gll_ft1_tr_bn190114873_v00_filt_prob.fit srcmdl=gll_ft1_tr_bn190114873_v00_filt_likeRes.xml irfs="P8R3_TRANSIENT020E_V2" evtype="INDEF" srclist= chatter=2 clobber=yes debug=no gui=no mode="ql"` 

As you can see this uses the P8R3_TRANSIENT020E_V2 IRF. Whereas, all the previous steps (exposure map calculation, gtdiffrsp, likelihood) all use P8R3_TRANSIENT020_V3.  For instance, here is the call to gtexpmap.
`/home/sajjad/miniconda2/envs/fermi_3ML/lib/python3.9/site-packages/fermitools/GtBurst/gtapps_mp/gtexpmap_mp.py 40 40 4 2 /home/sajjad/SOURCES_ASTRO/GRBs/THREE_ML/2022_10/bn190114873/gll_ft2_tr_bn190114873_v00.fit /home/sajjad/SOURCES_ASTRO/GRBs/THREE_ML/2022_10/interval2.6996-3.6358/gll_ft1_tr_bn190114873_v00_filt.fit gll_ft1_tr_bn190114873_v00_filt_ltcube.fit P8R3_TRANSIENT020E_V3 20.0 20 gll_ft1_tr_bn190114873_v00_filt_expomap.fit`

Could you let me know how I can correct this problem?

Thank you in advance!

**Comment by omodei** [comment]
Mmm. It looks like you are analyzing an old version of Fermi LAT data. I would suggest top entirely remove every data previously downloaded and start from scratch. Also, another option would be to try to run _gtburst_ by command line. It's the one starting with "doTimeResolvedLikelihood.py" followed by a  bunch of arguments. Copy paste and execute the command. let me know.

**Comment by ssajjadak** [comment]
Thank you . This was helpful. I had not thought that the old data would affect this analysis since I was working in a newly created directory and I could see that the threeML script was downloading the FSSC data again. But it seems like the code might be reading part of the data from the ~/FermiData/bn190114873/ directory that gtburst created when I worked with it previously.

So I first tried to work with a different GRB (one that I was sure of never having analysed before) and could get the code to work with it. After that, I tried to work with GRB 190114C again after deleting the ~/FermiData/bn190114873/ directory and I could get the code to work again. 

So it does seem like the old data in the ~/FermiData directory was the problem. Does this mean that threeML /gtburst automatically looks for data in the ~/FermiData/ directory and that we can not make it work with only the data in the current directory (or a user-defined directory)?
